### ISSUE TITLE
Feature: Email Event Participants

### ISSUE THREAD

**Issue by pikurasa** [bookingactivities/booking-activities#112]
It would be nice to be able to click on an event and send a (custom) email to all event participants.

Proposal:
- Click event booking from bookings calendar page
- Option to "Email Customers/Participants" in Actions section
- Sent to a new page with all customer email address in the "To" section
- Admin fills in a custom message for subject and body
- Email is sent when the admin clicks "send"

This feature would be very helpful. Right now, an administrator would need to go through a few steps to collect all emails. Plus, BA already has notification capabilities.

**Comment by yoancutillas** [comment]
Thank you, I had noted that feature, I add your vote :)

**Comment by pikurasa** [comment]
Is this similar to https://github.com/bookingactivities/booking-activities/issues/126 ? If so, please aggregate the votes.

**Comment by SophieWeb** [comment]
I think that with the release of Booking Activities 1.16 we are almost there.

Maybe what’s missing is the ability to create a « manual notification » template to be used when sending a message without changing booking status or triggering any other notification?

**Comment by yoancutillas** [comment]
Yes, the hardest part is done (sending a notification to all the participants of an event) with Booking Activities 1.16 bulk actions, so I think I can include a manual notification in the next Notification Pack release.
